content/concepts/configuration.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Configuration
+---
+
+## Overview 
+You may have noticed that few webpack configurations look exactly alike. This is because **webpack's configuration file is a JavaScript file that exports an object.** This object, is then parsed by webpack based upon its defined properties.
+
+### Examples
+The following examples below describe how webpacks configuration object can be both expressive and configurable because _it is code_: 
+
+#### The simplest of examples
+**webpack.config.js**
+```javascript 
+module.exports = {
+  entry: './foo.js',
+  output: {
+    path: 'dist',
+    filename: 'foo.bundle.js'
+  }
+}
+
+```
+
+#### Generating multiple targets and configs and exporting the entire array.
+**webpack.config.js**
+```javascript
+var path = require('path');
+var webpack = require('webpack');
+var webpackMerge = require('webpack-merge');
+
+var baseConfig = {  
+  target: 'async-node',
+  entry: {
+    entry: './entry.js'
+  },
+  output: {
+    filename: '[name].js',
+    path: path.resolve(__dirname, './dist')
+  },
+  plugins: [
+    new webpack.optimize.CommonsChunkPlugin({
+      name: 'inline',
+      filename: 'inline.js',
+      minChunks: Infinity
+    }),
+    new webpack.optimize.AggressiveSplittingPlugin({
+        minSize: 5000,
+        maxSize: 10000
+    }),
+  ]
+};
+
+let targets = ['web', 'webworker', 'node', 'async-node', 'node-webkit', 'electron-main'].map((target) => {
+  let base = webpackMerge(baseConfig, {
+    target: target,
+    output: {
+      path: path.resolve(__dirname, './dist/' + target),
+      filename: '[name].' + target + '.js'
+    }
+  });
+  return base;
+});
+
+module.exports = targets;
+```
+
+#### Using TypeScript to create a class which the angular-cli uses to [generate configs](https://github.com/angular/angular-cli/blob/master/addon/ng2/models/webpack-config.ts)
```

Comment:
This is a separate document called `configuration`. 

Essentially it gives the learner a bit of context WRT understand the consistent 'shape' (or seemingly lack there of) of the config.